src/redux/tasks.js

```diff
@@ -0,0 +1,41 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const taskSlice=createSlice({
+    name:'tasks',
+    initialState:[],
+    reducers:{
+        ADDTASK:(state,action)=>{
+           state=[...state,action.payload]
+           return state
+        },
+        REMOVETASK:(state,action)=>{
+            state=state.filter((e)=>e.id!==action.payload.id)
+            return state
+        },
```

Comment:
Use a better naming convention. As these are functions there names should be like addTask and so on.